src/types/supabase.ts

```diff
@@ -12,25 +12,31 @@ export interface Database {
       access_points_latest_states: {
```

Comment:
If we don't automate this, can we document somewhere (in project readme or src/types/readme.md?) how to update these types?

Reply:
OK, I'll do that as a separate PR